x/wasm/keeper/handler_plugin_encoders.go

```diff
@@ -31,25 +31,28 @@ type (
 
 type MessageEncoders struct {
 	Bank         func(sender sdk.AccAddress, msg *wasmvmtypes.BankMsg) ([]sdk.Msg, error)
-	Custom       func(sender sdk.AccAddress, msg json.RawMessage) ([]sdk.Msg, error)
+	Custom       func(ctx sdk.Context, accountKeeper types.AccountKeeper, sender sdk.AccAddress, msg json.RawMessage) ([]sdk.Msg, error)
```

Comment:
`Custom` is just function with minimal parameters. Instead of adding the account keeper here, you would set a custom encoder method:
```go
func MyCustomEncoder(accountKeeper types.AccountKeeper)func(ctx sdk.Context, accountKeeper types.AccountKeeper, sender sdk.AccAddress, msg json.RawMessage) ([]sdk.Msg, error){
	return func(ctx sdk.Context, accountKeeper types.AccountKeeper, sender sdk.AccAddress, msg json.RawMessage) ([]sdk.Msg, error) {
		acct: = accountKeeper.GetAccount(ctx, someAccount)
		// ...do impl
	}
}
```